osd/cpu_overloaded.json

```diff
@@ -0,0 +1,8 @@
+{
+ "severity": "Warning",
+ "service_name": "SREManualAction",
+ "cluster_uuid": "${CLUSTER_UUID}",
+ "summary": "Action required: CPU overloaded",
+ "description": "Your OpenShift Dedicated cluster does not have enough CPU resources and requires you to take action. You need to either reduce application load or increase worker nodes. See the following documentation for scaling up worker nodes: https://docs.openshift.com/dedicated/4/getting_started/scaling-your-cluster.html",
```

Comment:
Can we make this a little more generic (CPU + Memory)? I think just removing "CPU" will do it.

Reply:
Good idea